### PR TITLE
fix: Automation Bugs and minor enhancements

### LINKED ISSUE #3935: Automation: Release bugs/enhancements
- [ ] Change the title of the item from "Message Contains" to "Message Content"
- [ ] When updating the automation, the alert says "Automation rule added successfully"
- [ ] Update doesn't work - I updated a rule, changed the condition, and hit save. It didn't update the automation.

### PR DESCRIPTION
**Pull request by fayazara**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #3935 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nithindavid — APPROVED** [review]

**Comment by pranavrajs** [comment]
Screenshot 2022-02-09 at 2 36 45 PM

@fayaz Add Condition Button is throwing this error.

**Comment by fayazara** [comment]
@pranavrajs When did this occur exactly? I tried appending conditions and I didnt get the same error.

Maybe a specific `condition` caused this, what had you selected?

https://user-images.githubusercontent.com/15716057/153170946-f2b1183f-6c61-4da0-99a0-112b8cbe6db2.mp4

**Comment by pranavrajs** [comment]
@fayazara 

Event: message_created
Filter: Message Contents
Operator: contains

**Comment by fayazara** [comment]
> @fayazara
> 
> Event: message_created Filter: Message Contents Operator: contains

@pranavrajs Fixed this, it was occuring because automations was following a different format than the filters.

Also fixed another bug where the query_operator was coming blank in edit mode.

File: app/javascript/dashboard/routes/dashboard/settings/automation/EditAutomationRule.vue [hunk 1]
```diff
@@ -365,12 +365,32 @@ export default {
       }
     },
     appendNewCondition() {
-      this.automation.conditions.push({
-        attribute_key: 'status',
-        filter_operator: 'equal_to',
-        values: '',
-        query_operator: 'and',
-      });
+      if (
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/settings/automation/EditAutomationRule.vue:368]: Can we refactor into 

```
const { conditions } = this.automation;
const [lastCOndition] = conditions;
const lastQueryOperator = lastCOndition.query_operator
```

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Review by muhsin-k — DISMISSED** [review]

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**